mesa/space.py

```diff
@@ -504,6 +504,14 @@ def position_agent(
         If x or y are positive, they are used.
         Use 'swap_pos()' to swap agents positions.
         """
+        warn(
+            (
+                "`position_agent` is being deprecated in favor of "
+                "`place_agent` and `move_to_empty`."
```

Comment:
I think the previous warning message describing when to use `place_agent` and `move_to_empty` is better.

Reply:
new way :-)